org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -0,0 +1,171 @@
+Parameters:
```

Comment:
Is this an org-formation config' file?  What would I read up on to have the knowledge to review this?

Reply:
yes, it's an org-formation template.  docs for org-formation are here https://github.com/org-formation/org-formation-cli

the example templates to setup SSO used for this PR are here  https://github.com/org-formation/org-formation-reference/tree/master/src/templates/100-aws-sso